test/core/ext/filters/rbac/rbac_service_config_parser_test.cc

```diff
@@ -738,10 +742,13 @@ TEST_F(RbacServiceConfigParsingTest, AuditConditionOnDenyWithMultipleLoggers) {
   ASSERT_NE(parsed_rbac_config->authorization_engine(0), nullptr);
   EXPECT_EQ(parsed_rbac_config->authorization_engine(0)->audit_condition(),
             Rbac::AuditCondition::kOnDeny);
-  EXPECT_EQ(parsed_rbac_config->authorization_engine(0)->num_audit_loggers(),
-            2);
-  ASSERT_NE(logger_configs_.find("test_logger"), logger_configs_.end());
-  EXPECT_EQ(logger_configs_.find("test_logger")->second, "{\"foo\":\"bar\"}");
+  const auto& loggers =
+      parsed_rbac_config->authorization_engine(0)->audit_loggers();
+  ASSERT_EQ(loggers.size(), 2);
```

Comment:
Suggest writing this as:

```
EXPECT_THAT(
    loggers
    ::testing::ElementsAre(
        ::testing::Property(&AuditLogger::name, "stdout_logger"),
        ::testing::Property(&AuditLogger::name, kLoggerName)));
```

Reply:
Done. I had to use
```
  EXPECT_THAT(parsed_rbac_config->authorization_engine(0)->audit_loggers(),
              ::testing::ElementsAre(::testing::Pointee(::testing::Property(
                                         &AuditLogger::name, "stdout_logger")),
                                     ::testing::Pointee(::testing::Property(
                                         &AuditLogger::name, kLoggerName))));
```
because the vector stores unique pointers.